typescript/src/client.ts

```diff
@@ -0,0 +1,25 @@
+import { Connection, Client } from '@temporalio/client';
+import { breakfastWorkflow } from './workflows';
+import { nanoid } from 'nanoid';
+
+async function run() {
+  // Connect to the default Server location (localhost:7233)
+  const connection = await Connection.connect();
+
+  const client = new Client({
+    connection,
+  });
+
+  const handle = await client.workflow.start(breakfastWorkflow, {
+    args: [],
+    taskQueue: 'make-breakfast',
```

Comment:
FWIW, I appreciate that this uses a string literal for the task queue where needed and I think it makes sense to do that in the other samples